server/api/server_test.go

```diff
@@ -117,3 +142,47 @@ func mustWaitLeader(c *C, svrs []*server.Server) *server.Server {
 	c.Fatal("no leader")
 	return nil
 }
+
+func mustRPCRequest(c *C, addr string, request *pdpb.Request) *pdpb.Response {
```

Comment:
Why not use https://github.com/pingcap/pd/blob/c77e51541f7c9df54af3586d13d377240e4f9a01/pkg/testutil/testutil.go#L36